### DISCUSSION TITLE
Save macros to execute commands

### DISCUSSION THREAD

**Discussion by josepcorrea** [Eugeny/tabby#6446] [General]
Is there any option to save a macro, in which, accessing by ssh to a server, you can execute predefined commands? Thanks.

**Comment by Eugeny** [comment]
Yes - see the quick-cmds plugin.

**Comment by FlyingT** [comment]
Since this is the first thing that comes up in Google, this is the plugin:
https://github.com/minyoad/tabby-quick-cmds

You can install it in Tabby under Settings > Extensions and scroll down to quick-cmds
